protocols/relay/src/lib.rs

```diff
@@ -23,9 +23,37 @@
 
 #![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
 
+pub mod behaviour;
```

Comment:
```suggestion
mod behaviour;
```

Reply:
thanks, in the process I made `CircuitId` pub on `relay::CircuitId` as it was already `pub`